From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@kenrward__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
removing label to test repo automation

PRmerger commented:
@kenrward : Thanks for your contribution to the Azure documentation! The author, @adamglick, has been notified to review your proposed change.

adamglick commented:
Hi @kenrward thanks for the update!  In this case I think there may be some confusion as the "Cloud Services" listed here are for the actual service offering of Azure Cloud Services.  I'd be happy to link to a doc on designing for use of Cloud Services (PaaS) though the link included (the Failsafe content) is out of date, has a number of broken links and deals with the general design of cloud services. I do know that Khalid (the author of the DR doc) is looking into making one that will be more design focused and service specific. Do you feel the Failsafe content is Azure Cloud Services (PaaS) specific to designing for HA (in general the PaaS services don't need HA design ans that's what a PaaS is supposed to do for users though I'm not deep on Cloud Services so this may be an exception)..

kenrward commented:
I think we need to link something or change the wording "not needed" is better than "not available".  Also, what about linking to Availability Set documentation for VM's?

kenrward commented:
Sorry, meant to "close & comment"

codemillmatt committed:
Merge remote-tracking branch 'MicrosoftDocs/master'

codemillmatt committed:
Adding Xamarin QuickStart to index

codemillmatt committed:
Merge remote-tracking branch 'MicrosoftDocs/master' into xam-quickstart-cosmos-index

codemillmatt committed:
Typo in link!

cephalin committed:
refactor easy auth

cephalin committed:
acrolinx

cephalin committed:
staged review

ccompy committed:
update addresses

removed geo-region IPs and added Fairfax

CarlRabeler committed:
[sql db] bucharest

CarlRabeler committed:
more

CarlRabeler committed:
more

CarlRabeler committed:
Merge branch 'master' into bucharest

rolyon committed:
Updates to use CLI for custom roles

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-externusers-cli

rolyon committed:
Additional fixes based on Acrolinx

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-externusers-cli

Commit:
updates for the rebranding of OMS

Commit:
updates for the rebranding of OMS

CarlRabeler committed:
read scale-oout

CarlRabeler committed:
more

ccompy committed:
update

ccompy committed:
update

CarlRabeler committed:
more

CarlRabeler committed:
period

cephalin committed:
review feedback

cephalin committed:
nits

codemillmatt committed:
Merge remote-tracking branch 'MicrosoftDocs/master' into xam-quickstart-cosmos-index

codemillmatt committed:
Updating the Xamarin QS

codemillmatt committed:
Merge remote-tracking branch 'MicrosoftDocs/master' into xam-quickstart-cosmos-index

tanmayeekamath committed:
Add DevTest Labs

tanmayeekamath committed:
Create documentation-government-services-devtools.md

tanmayeekamath committed:
Added DevTools Labs TOC

CarlRabeler committed:
dtus

CarlRabeler committed:
Merge branch 'master' into bucharest

CarlRabeler committed:
links

CarlRabeler committed:
links

AdiGanMSFT committed:
Updating SQL enterprise

codemillmatt committed:
Finishing off getting started article

codemillmatt committed:
Merge remote-tracking branch 'MicrosoftDocs/master' into xam-quickstart-cosmos-index

codemillmatt committed:
Image borders

CarlRabeler committed:
sasha

CarlRabeler committed:
acro - service tiers

CarlRabeler committed:
more

CarlRabeler committed:
table

CarlRabeler committed:
in-memory

CarlRabeler committed:
minor

CarlRabeler committed:
cores etc

CarlRabeler committed:
more

CarlRabeler committed:
links

codemillmatt committed:
Merge remote-tracking branch 'MicrosoftDocs/master' into xam-quickstart-cosmos-index

CarlRabeler committed:
size increments

CarlRabeler committed:
resource limits

CarlRabeler committed:
resource limits

CarlRabeler committed:
ltr draft

CarlRabeler committed:
temporarily restore resource limits file

CarlRabeler committed:
delete

CarlRabeler committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into bucharest

codemillmatt committed:
Updates as per review

CarlRabeler committed:
more

paulathurman committed:
edit pass: hana-overview-architecture

paulathurman committed:
edit pass: hana-overview-architecture

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-craic0320b

CarlRabeler committed:
ready for review

Commit:
updates after review

CarlRabeler committed:
per review

CarlRabeler committed:
sasha edits

codemillmatt committed:
Merge remote-tracking branch 'MicrosoftDocs/master' into xam-quickstart-cosmos-index

ccompy committed:
title fix

ccompy committed:
fixed the title

ccompy committed:
updated creating in a tunneled vnet

codemillmatt committed:
Fixes for PR #36361

codemillmatt committed:
Merge remote-tracking branch 'MicrosoftDocs/master' into xam-quickstart-cosmos-index

ccompy committed:
more updates

ccompy committed:
doc update

ccompy committed:
doc update

ccompy committed:
tweak

ccompy committed:
date

ccompy committed:
date

ccompy committed:
date

ccompy committed:
added image

ccompy committed:
new file

ccompy committed:
upload file

ccompy committed:
upload file

ccompy committed:
one more time

tanmayeekamath committed:
Updates Variations and Limitations for DTL

gsacavdm committed:
Fix links in gov-detvtools

ccompy committed:
tweak

ccompy committed:
fixed typos

ccompy committed:
fixed typo

liwong88 committed:
added breadcrumbs

liwong88 committed:
updated link

Commit:
MVC Virtual Network

Commit:
Added period

codemillmatt committed:
Merge remote-tracking branch 'MicrosoftDocs/master' into xam-quickstart-cosmos-index

codemillmatt committed:
Adding instructions on how to run the app

CarlRabeler committed:
table

CarlRabeler committed:
no IO

CarlRabeler committed:
faq

CarlRabeler committed:
more

CarlRabeler committed:
more

Commit:
updates for the rebranding of OMS

CarlRabeler committed:
more FAQ

CarlRabeler committed:
per PR review

ajaycode committed:
FAQ on NPM pricing

ajaycode committed:
changes based on acrolinx feedback

ajaycode committed:
formatted table

ajaycode committed:
removed abbreviated mentions of PM

ajaycode committed:
right aligned data in table

CarlRabeler committed:
vcores and dtus

CarlRabeler committed:
more

ajaycode committed:
metadata fixes

ajaycode committed:
Update log-analytics-network-performance-monitor-pricing-faq.md

matt1883 committed:
Correcting ADLA/S account creation snippets in ADLA python article

CarlRabeler committed:
ltr configure

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-craic0330

CarlRabeler committed:
link

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into mvc-virtual-network-new

CarlRabeler committed:
links

CarlRabeler committed:
toc

CarlRabeler committed:
rename

CarlRabeler committed:
toc

CarlRabeler committed:
toc

CarlRabeler committed:
links

CarlRabeler committed:
more

carolinacmoravia committed:
L14965: Double square bracket insterted breaking hyperlink 

Hello, @kgremban,

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version. 

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

CarlRabeler committed:
images

CarlRabeler committed:
more

CarlRabeler committed:
more

CarlRabeler committed:
table

ccompy committed:
incorporated feedback

ccompy committed:
app gw update

ccompy committed:
removed powerapps in the title

fixed stuff

CarlRabeler committed:
review edits

ccompy committed:
tweak

Commit:
Removed internal links

Commit:
RTMP spec update

mimckitt committed:
Fixes MicrosoftDocs/azure-docs#6657

Fixed typo. The word scripts was spelt as scipts in two places.

PRMerger13 committed:
Merge pull request #36951 from johndeu/master

Removed internal links

PRMerger13 committed:
Merge pull request #36953 from MicahMcKittrick-MSFT/patch-15

Fixes MicrosoftDocs/azure-docs#6657

CarlRabeler committed:
sasha edits

CarlRabeler committed:
typo

Commit:
Update analysis-services-gateway.md

Commit:
Update analysis-services-gateway-install.md

GitHubber17 committed:
Merge pull request #36775 from ccompy/mar14

update of several docs

GitHubber17 committed:
Merge pull request #36811 from lisawong19/cog-knowledge

added breadcrumbs

Commit:
Update analysis-services-import-pbix.md

Christina-Kang committed:
Update service-fabric-health-introduction.md

Typo fix.

dlepow committed:
N-series: Updates for GRID 6.0 drivers

dlepow committed:
metadata update

PRMerger13 committed:
Merge pull request #36957 from Minewiskan/patch-2

Update analysis-services-gateway-install.md

gatneil committed:
quick factual fixes

PRMerger13 committed:
Merge pull request #36956 from Minewiskan/patch-1

Update analysis-services-gateway.md

PRMerger15 committed:
Merge pull request #36958 from Minewiskan/patch-3

Update analysis-services-import-pbix.md

vladvino committed:
Updated OpenAPI import limitations

rmca14 committed:
Merge pull request #35725 from CarlRabeler/bucharest

[sql db] bucharest

PRMerger19 committed:
Merge pull request #36962 from vladvino/master

Updated OpenAPI import limitations

tdykstra committed:
change runonstartup text

gunjanj committed:
Updated regions availability

Removed west central, added Canada east

kgremban committed:
Update LBS quickstart

PRMerger14 committed:
Merge pull request #36967 from kgremban/apr3-lbsquickstart

Update LBS quickstart

PRMerger15 committed:
Merge pull request #36961 from gatneil/quickfix

quick factual fixes

ShawnJackson committed:
Update hana-overview-architecture.md

CarlRabeler committed:
Update sql-database-long-term-backup-retention-configure.md

CarlRabeler committed:
Update sql-database-long-term-backup-retention-configure.md

ShawnJackson committed:
Merge pull request #36562 from paulathurman/hana-overview-architecture

edit pass: hana-overview-architecture

Commit:
Merge pull request #36968 from CarlRabeler/patch-41

Update sql-database-long-term-backup-retention-configure.md

Commit:
Merge pull request #36970 from CarlRabeler/patch-42

Update sql-database-long-term-backup-retention-configure.md

christiankuhtz committed:
tweak

PRMerger12 committed:
Merge pull request #36973 from christiankuhtz/patch-116

tweak

rolyon committed:
Updates to the custom roles section for PowerShell and CLI

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-externusers-cli

BryanTrach-MSFT committed:
fixes MicrosoftDocs/azure-docs#6653

Updated the create user link in the document with below link. Previous link was causing a 404 error.

https://docs.microsoft.com/en-us/rest/api/apimanagement/user/createorupdate

Juliako committed:
Update media-services-portal-publish.md

PRMerger17 committed:
Merge pull request #36975 from BryanTrach-MSFT/patch-11

fixes MicrosoftDocs/azure-docs#6653

PRMerger17 committed:
Merge pull request #36977 from Juliako/patch-3

Update media-services-portal-publish.md

Commit:
Merge pull request #36959 from Christina-Kang/patch-2

Update service-fabric-health-introduction.md

Commit:
Merge pull request #36966 from gunjanj/patch-11

Updated regions availability

GitHubber17 committed:
Merge pull request #36799 from tanmayeekamath/patch-1

Add DevTest Labs to Azure Gov

derek1ee committed:
Update scheduler-plans-billing.md

rolyon committed:
Deleted dataActions refs, restored heading to fix link

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-externusers-cli

PRMerger19 committed:
Merge pull request #36960 from dlepow/gpu

N-series: Updates for GRID 6.0 drivers

ajaycode committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into network-monitoring-overview

ajaycode committed:
Updated ToC

rwike77 committed:
Freshness

ajaycode committed:
changed H1

PRMerger10 committed:
Merge pull request #36980 from derek1ee/patch-5

Update scheduler-plans-billing.md

rwike77 committed:
Fixed date

PRMerger20 committed:
Merge pull request #36983 from rwike77/patch-1

Freshness

ajaycode committed:
changed H1

ajaycode committed:
meta data changes

ajaycode committed:
reasons for variations in data volume

cephalin committed:
prmerger feedback

ajaycode committed:
introduction

ajaycode committed:
Service Endpoint Monitor

PRMerger17 committed:
Merge pull request #36341 from AdiGanMsft/patch-10

Updating SQL SKU

ajaycode committed:
language refinement

codemillmatt committed:
Merge remote-tracking branch 'MicrosoftDocs/master' into xam-quickstart-cosmos-index

Commit:
Merge pull request #36926 from matt1883/patch-3

Correcting ADLA/S account creation snippets in ADLA python article

codemillmatt committed:
Updating first screen images and text

codemillmatt committed:
Merge remote-tracking branch 'MicrosoftDocs/master' into xam-quickstart-cosmos-index

Commit:
TOC update.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into mvc-virtual-network-new

PRMerger12 committed:
Merge pull request #36963 from tdykstra/patch-31

change runonstartup text

codemillmatt committed:
the PNG file

carolinacmoravia committed:
L15412: Extra double quote causing processing issues with the MS transformer

Hello, @iainfoulds, 

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

Brenduns committed:
Fixing a link that looks fine, but rendered bad

Commit:
updates after review comments

PRMerger12 committed:
Merge pull request #36937 from carolinacmoravia/patch-657

L14965: Double square bracket insterted breaking hyperlink

PRMerger14 committed:
Merge pull request #36996 from Brenduns/update-1803-notes

Fixing a link that looks fine, but rendered bad

MGoedtel committed:
Updated statement about circular logging

MGoedtel committed:
Updated statement about circular logging

codemillmatt committed:
Removing shadows - adding note

Commit:
Merge pull request #36997 from MGoedtel/UpdateLACustLogs44

Updated statement about circular logging

itechedit committed:
Merge pull request #35374 from cephalin/security

refactor easy auth

CarlRabeler committed:
Update sql-database-long-term-backup-retention-configure.md

Brenduns committed:
New update issue

CarlRabeler committed:
Update sql-database-faq.md

CarlRabeler committed:
Update sql-database-long-term-backup-retention-configure.md

itechedit committed:
Merge pull request #36361 from codemillmatt/xam-quickstart-cosmos-index

New SQL API Xamarin QuickStart

CarlRabeler committed:
Update sql-database-long-term-backup-retention-configure.md

PRMerger15 committed:
Merge pull request #36998 from CarlRabeler/patch-43

Update sql-database-long-term-backup-retention-configure.md

PRMerger16 committed:
Merge pull request #36999 from Brenduns/update-1803-notes

New update issue

PRMerger16 committed:
Merge pull request #37000 from CarlRabeler/patch-44

Update sql-database-faq.md

CarlRabeler committed:
Merge branch 'master' into patch-45

PRMerger18 committed:
Merge pull request #37002 from CarlRabeler/patch-46

Update sql-database-long-term-backup-retention-configure.md

itechedit committed:
Merge pull request #35910 from craigcaseyMSFT/v-craic0320b

updates for the rebranding of OMS

PRMerger19 committed:
Merge pull request #37001 from CarlRabeler/patch-45

Update sql-database-long-term-backup-retention-configure.md

PRMerger20 committed:
Merge pull request #36994 from carolinacmoravia/patch-658

L15412: Extra double quote causing processing issues with the MS tran…

itechedit committed:
Merge pull request #35876 from rolyon/rolyon-externusers-cli

[Azure AD RBAC]  Updates to use CLI 2.0 for custom roles

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

rmca14 committed:
Merge pull request #36814 from jimdial/mvc-virtual-network-new

MVC Virtual Network

itechedit committed:
Merge pull request #36858 from craigcaseyMSFT/v-craic0330

updates for the rebranding of OMS

v-alje committed:
Merge pull request #37006 from MicrosoftDocs/FromPublicRepo

From public repo

itechedit committed:
Merge pull request #36895 from ajaycode/network-monitoring-overview

FAQ on NPM pricing
